src/app/api/schemas.py

```diff
@@ -36,7 +36,7 @@ class UserRead(UserInfo, _CreatedAt):
 
 # Authentication request
 class UserAuth(UserInfo, Cred):
-    scopes: Optional[str] = "me"
+    scopes: str = Field("me", example="me")
```

Comment:
I think the `Optional[str]`should be preserved here?

Reply:
As far as I have understood, adding elements instead of ... in field make it optionnal.
https://pydantic-docs.helpmanual.io/usage/schema/

I have tested (I can join screens later), but you can test by yourself inserting some user without scope.

Reply:
Oh my bad then!